imgui/src/io.rs

```diff
@@ -447,92 +485,114 @@ fn test_io_memory_layout() {
-    assert_field_offset!(ime_window_handle, ImeWindowHandle);
-    assert_field_offset!(mouse_pos, MousePos);
-    assert_field_offset!(mouse_down, MouseDown);
-    assert_field_offset!(mouse_wheel, MouseWheel);
-    assert_field_offset!(mouse_wheel_h, MouseWheelH);
-    assert_field_offset!(key_ctrl, KeyCtrl);
-    assert_field_offset!(key_shift, KeyShift);
-    assert_field_offset!(key_alt, KeyAlt);
-    assert_field_offset!(key_super, KeySuper);
-    assert_field_offset!(keys_down, KeysDown);
-    assert_field_offset!(nav_inputs, NavInputs);
-    assert_field_offset!(want_capture_mouse, WantCaptureMouse);
-    assert_field_offset!(want_capture_keyboard, WantCaptureKeyboard);
-    assert_field_offset!(want_text_input, WantTextInput);
-    assert_field_offset!(want_set_mouse_pos, WantSetMousePos);
-    assert_field_offset!(want_save_ini_settings, WantSaveIniSettings);
-    assert_field_offset!(nav_active, NavActive);
-    assert_field_offset!(nav_visible, NavVisible);
-    assert_field_offset!(framerate, Framerate);
-    assert_field_offset!(metrics_render_vertices, MetricsRenderVertices);
-    assert_field_offset!(metrics_render_indices, MetricsRenderIndices);
-    assert_field_offset!(metrics_render_windows, MetricsRenderWindows);
-    assert_field_offset!(metrics_active_windows, MetricsActiveWindows);
-    assert_field_offset!(metrics_active_allocations, MetricsActiveAllocations);
-    assert_field_offset!(mouse_delta, MouseDelta);
-    assert_field_offset!(key_mods, KeyMods);
-    assert_field_offset!(key_mods_prev, KeyModsPrev);
-    assert_field_offset!(mouse_pos_prev, MousePosPrev);
-    assert_field_offset!(mouse_clicked_pos, MouseClickedPos);
-    assert_field_offset!(mouse_clicked_time, MouseClickedTime);
-    assert_field_offset!(mouse_clicked, MouseClicked);
-    assert_field_offset!(mouse_double_clicked, MouseDoubleClicked);
-    assert_field_offset!(mouse_clicked_count, MouseClickedCount);
-    assert_field_offset!(mouse_clicked_last_count, MouseClickedLastCount);
-    assert_field_offset!(mouse_released, MouseReleased);
-    assert_field_offset!(mouse_down_owned, MouseDownOwned);
-    assert_field_offset!(mouse_down_duration, MouseDownDuration);
-    assert_field_offset!(mouse_down_duration_prev, MouseDownDurationPrev);
-    assert_field_offset!(mouse_drag_max_distance_abs, MouseDragMaxDistanceAbs);
-    assert_field_offset!(mouse_drag_max_distance_sqr, MouseDragMaxDistanceSqr);
-    assert_field_offset!(keys_down_duration, KeysDownDuration);
-    assert_field_offset!(keys_down_duration_prev, KeysDownDurationPrev);
-    assert_field_offset!(nav_inputs_down_duration, NavInputsDownDuration);
-    assert_field_offset!(nav_inputs_down_duration_prev, NavInputsDownDurationPrev);
-    assert_field_offset!(pen_pressure, PenPressure);
-    assert_field_offset!(input_queue_surrogate, InputQueueSurrogate);
-    assert_field_offset!(input_queue_characters, InputQueueCharacters);
+
+    std::thread::Builder::new()
```

Comment:
Maybe worth short comment explaining why the thread, and maybe a link to https://github.com/rust-lang/rfcs/pull/3308